frontend-react/src/__mocks__/DBMock.ts

```diff
@@ -0,0 +1,20 @@
+import { Faker } from "../utils/Faker";
+import { createDbMock } from "../utils/MSWData/CreateDbMock";
+
+import { createSettingsModels } from "./SettingsMock";
+
+export function createModels(faker: Faker) {
+    return {
+        ...createSettingsModels(faker),
```

Comment:
I realize we've already talked about this, but I do think it's worth aligning with the singleton pattern that Faker recommends for normal use cases.  After configuration/initialization, we shouldn't be mutating the instance in test cases, and it'd benefit us by not having to instantiate new Faker instances with extended options each time we want to invoke one of these functions.  If we want the ability to pass in a separate Faker instance to each of these functions, I'd at least recommend defaulting the `faker` value to be the default Faker instance created [here](https://github.com/CDCgov/prime-reportstream/pull/8815/files#diff-6ffc1dec4e79ad72a998f64e278625d0e52cd1b2fd098a0498daf00c04afddafR161).  What are your thoughts about this?